radio/src/opentx.cpp

```diff
@@ -2572,7 +2572,7 @@ int main()
   // lcdSetRefVolt(25);
 #endif
 
-#if defined(PCBTARANIS) || defined(PCBHORUS)
+#if defined(SPLASH) && (defined(PCBTARANIS) || defined(PCBHORUS))
```

Comment:
I'm almost positive that I noticed the error before I merged the latest `next` commits which included your change.  I remember noticing your commit note and wondering if perhaps that had fixed the issue.  Then again it's been a long week...  :)